src/router/kromo/dd-wrt/elegant/fresh-dark.css

```diff
@@ -137,7 +137,7 @@ select {
 select[disabled] {
   color: #666;
   background-color: #1a1a1a;
-  border-color: #111111;
+  border-color: #111;
 }
```

Comment:
You dont need to write out 6 numbers in some cases, 3 is enough. I dont know how many bits its saves, but if you do this often enough it matters.

The browsers can read and understand this and besides, its inconsistent to mix short hex and long hex on same file and mix lowercase and  uppercase and if like me your eyes are perfect, its hard to read hex values where `8B` appear together, so lowercase helps me a great deal in general.